README.md

```diff
@@ -0,0 +1,16 @@
+# BTrust Builders Discussion Questions Repository
+
+Welcome to the **BTrust Builders** GitHub repository for discussion questions related to the following courses. This repository is designed to facilitate meaningful discussions and enhance your understanding of key concepts.
+
+1. **Mastering Bitcoin - Discussion Questions**
+   - Explore and engage in discussions here : [Mastering Bitcoin Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-bitcoin/README.md)
+
+2. **Mastering the Lightning Network - Discussion Questions**
+   - Explore and engage in discussions here: [Mastering the Lightning Network Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-lightning/README.md)
+
+
+**NB:** An overview of the questions for which you will be the discussion leader will be shared over the usual channels. Daily discussions will be scheduled on the calendar.
```

Comment:
Perhaps include a link to where people can learn more about discussions? If someone stumbles on this repo they may be interested in participating with a group.